db/migration/src/m20240226_000002_create_ticket.rs

```diff
@@ -0,0 +1,73 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_table(
+                Table::create()
+                    .table(Ticket::Table)
+                    .if_not_exists()
+                    .col(
+                        ColumnDef::new(Ticket::Id)
+                            .integer()
+                            .not_null()
+                            .auto_increment()
+                            .primary_key(),
+                    )
+                    .col(ColumnDef::new(Ticket::ChannelId).string_len(64).not_null())
+                    .col(ColumnDef::new(Ticket::Amount).string_len(50).not_null())
+                    .col(ColumnDef::new(Ticket::Index).integer().unsigned().not_null())
+                    .col(ColumnDef::new(Ticket::IndexOffset).integer().unsigned().not_null())
+                    .col(ColumnDef::new(Ticket::WinningProbability).binary_len(7).not_null())
+                    .col(
+                        ColumnDef::new(Ticket::ChannelEpoch)
+                            .integer()
+                            .unsigned()
+                            .not_null()
+                            .default(1),
+                    )
+                    .col(ColumnDef::new(Ticket::EthereumChallenge).binary_len(64).not_null())
+                    .col(ColumnDef::new(Ticket::Signature).binary_len(60).not_null())
+                    .col(ColumnDef::new(Ticket::AcknowledgementData).binary().not_null())
+                    .foreign_key(
+                        ForeignKey::create()
+                            .name("fk_ticket_channel")
```

Comment:
Does this generate an index automatically? Otherwise the channel id references could use index coverage.

Reply:
Yes FKs generate indices automatically